src/amo/containers/App.js

```diff
@@ -20,9 +34,22 @@ export class AppBase extends React.Component {
           defaultTitle={i18n.gettext('Add-ons for Firefox')}
         />
         {children}
+        <footer>{this.logIn()}</footer>
       </div>
     );
   }
 }
 
-export default translate({ withRef: true })(AppBase);
+export const makeMapStateToProps = ({ _window } = {}) => (state) => ({
+  isAuthenticated: !!state.auth.token,
+  handleLogIn() {
+    // eslint-disable-next-line no-param-reassign
+    (_window || window).location = startLoginUrl();
```

Comment:
I mean, it seems like something we won't need server side, just wondering if something might call it somewhere